frontend/components/student/toursList.tsx

```diff
@@ -1,49 +1,67 @@
-import { StudentOnTour } from "@/lib/student-on-tour";
-import { Tour } from "@/lib/tour";
-import { RegionInterface } from "@/lib/region";
-import { ListGroup, ListGroupItem } from "react-bootstrap";
+import {StudentOnTour} from "@/lib/student-on-tour";
+import {Tour} from "@/lib/tour";
+import {RegionInterface} from "@/lib/region";
+import {ListGroup, ListGroupItem} from "react-bootstrap";
 
 /**
  * A component for the list of the tours a student is/was assigned to
  */
 export default function ToursList({
-    listTitle,
-    studentOnTours,
-    allTours,
-    allRegions,
-    onSelect,
-}: {
+                                      listTitle,
+                                      studentOnTours,
+                                      allTours,
+                                      allRegions,
+                                      onSelect,
+                                  }: {
     listTitle: String;
     studentOnTours: StudentOnTour[];
     allTours: Record<number, Tour>;
     allRegions: Record<number, RegionInterface>;
     onSelect: (studentOnTourId: number) => void;
 }) {
+
+    function getColor(sot : StudentOnTour) {
+        if (sot.completed_tour) {
```

Comment:
> Since I've completed this the tour of today and it still remains orange. And the tours that are completed don't seem to color red.

The tour is not marked as completed since the `completed` attribute isn't set when saving the last `VE` remark (this is in pr #450). I will change the coloring.

Reply:
Right good call.

Reply:
I added the temporary end-call.